doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -306,6 +306,8 @@ The requirements in this section are to be delivered in addition to those in [se
 | `req.inf.ntw.10` | Infrastructure | Network | The Architecture **must** support Cluster wide coordination of IP address assignment. | |
 | `req.inf.ntw.13` | Infrastructure | Network | The platform **must** allow specifying multiple separate IP pools. Tenants are required to select at least one IP pool that is different from the control infrastructure IP pool or other tenant IP pools. | |
 | `req.inf.ntw.14` | Infrastructure | Network | The platform **must** allow NATless traffic (i.e. exposing the pod IP address directly to the outside), allowing source and destination IP addresses to be preserved in the traffic headers from workloads to external networks. This is needed e.g. for signaling applications, using SIP and Diameter protocols.|[ra2.ntw.011](chapter04.md#45-networking-solutions)|
+| `req.inf.ntw.15` | Infrastructure | Network | The platform **must** support LoadBalancer [Publishing Service (ServiceType)](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) and [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/).||
```

Comment:
why a compound requirement? Can we split it into two requirements?

Reply:
Sure, I was just a bit lazy :)

Reply:
Fixed in 8acd2e1